packages/react-ui-core/src/Map/Map.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import mapboxgl from 'mapbox-gl/dist/mapbox-gl'
+
+export default class Mapbox extends Component {
+  static propTypes = {
+    size: PropTypes.string,
+    color: PropTypes.string,
+    theme: PropTypes.object,
+    token: PropTypes.string,
+    center: PropTypes.array,
+    style: PropTypes.string,
+    zoom: PropTypes.number,
+    theme: PropTypes.object,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  componentDidMount() {
+    mapboxgl.accessToken = this.props.token
+    const map = new mapboxgl.Map({
+      container: 'map',
```

Comment:
Can we make the `container` selector configurable via prop in case we ever need to render multiple maps on a page.